debezium-connector-postgres/RDS.md

```diff
@@ -0,0 +1,96 @@
+In order to run the test suite against an Amazon RDS PostgreSQL environment, there are several important steps that must be done in order for Debezium features to operate correctly on RDS and to allow remote connectivity to your Amazon Virtual Private Cloud (VPC) environment.
+
+## Setup remote connectivity
+
+When you setup an account with Amazon RDS, they should automatically create a new Security Group for your VPC environment.  By default this new security group does not allow remote connections from your local machine to the resources which it secures, so you you need to explicitly grant remote access.
```

Comment:
Could you use the "one sentence / unit-of-thought per line style". This avoids super-long lines while also not having to do hard breaks after a fixed number of characters. Diffs will be easier to view down the road that way.